packages/auth/src/graphql/resolvers/grant.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  ResolversTypes,
+  MutationResolvers,
+  Grant as SchemaGrant,
+  Access as SchemaAccess,
+  QueryResolvers
+} from '../generated/graphql'
+import { ApolloContext } from '../../app'
+import { Grant } from '../../grant/model'
+import { Pagination } from '../../shared/baseModel'
+import { getPageInfo } from '../../shared/pagination'
+import { Access } from '../../access/model'
+
+export const getGrants: QueryResolvers<ApolloContext>['grants'] = async (
+  _,
+  args,
+  ctx
+): Promise<ResolversTypes['GrantsConnection']> => {
+  const grantService = await ctx.container.use('grantService')
+  const grants = await grantService.getPage(args.input)
+  const pageInfo = await getPageInfo(
+    (pagination: Pagination) => grantService.getPage(pagination),
+    grants
+  )
+
+  return {
+    pageInfo,
+    edges: grants.map((grant: Grant) => ({
+      cursor: grant.id,
+      node: grantToGraphql(grant)
+    }))
+  }
+}
+
+export const revokeGrant: MutationResolvers<ApolloContext>['revokeGrant'] =
+  async (
+    _,
+    args,
+    ctx
+  ): Promise<ResolversTypes['RevokeGrantMutationResponse']> => {
+    try {
+      const { grantId } = args.input
+      if (!grantId) {
+        return {
+          code: '401',
+          success: false,
+          message: 'Grant Id is not provided'
+        }
+      }
```

Comment:
I think a "missing required argument" error will be thrown before reaching this (based on [this](https://github.com/interledger/rafiki/pull/1013/files#diff-216f1f943bb253f72501975d0646676f2526aaed362b1c582a77e06f7cc068fbR44)), but its not a big deal to keep here